docker/openemr/7.0.2/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apk add --no-cache git build-base libffi-dev python3-dev cargo \
     && git clone https://github.com/openemr/openemr.git --depth 1 \
     && rm -rf openemr/.git \
     && cd openemr \
-    && composer install --no-dev \
+    && composer install --no-dev --optimize-autoloader --apcu-autoloader \
```

Comment:
really cool stuff! nice read on it here:
https://getcomposer.org/doc/articles/autoloader-optimization.md

Looks like above command parameter and the below dump-autoload command are redundant. It only needs to be in the dump-autoload below, so would keep the parameters there, but no reason to have in above line (just in case for some reason we need to change/modify them int he future, best to have them in only one place).

Reply:
Made the change! Thanks for pointing that out!